main/src/main/scala/sbt/Opts.scala

```diff
@@ -52,14 +52,14 @@ object DefaultOptions {
   def pluginResolvers(plugin: Boolean, snapshot: Boolean): Seq[Resolver] = {
     if (plugin && snapshot) Seq(Classpaths.typesafeSnapshots, Classpaths.sbtPluginSnapshots) else Nil
   }
-  def addResolvers: Setting[_] = Keys.resolvers <++= Keys.isSnapshot apply resolvers
-  def addPluginResolvers: Setting[_] = Keys.resolvers <++= (Keys.sbtPlugin, Keys.isSnapshot) apply pluginResolvers
```

Comment:
removing this is a breaking change: https://github.com/scala/community-builds/pull/290